### ISSUE TITLE
End script back on branch/dettached state the the repo started on

### ISSUE THREAD

**Issue by pbeeson** [davidraviv/gh-clean-branches#6]
When `gh cleanup-branches` ends, it leaves the repo on branch `main`.  It would be nice to have the repo go back to the working branch used when the script was run.

I believe this is easily achievable by taking the current branch before changing to main (or the default branch like in PR #5)
```bash
branch=`git branch | sed -n '/\* /s///p' | rev | cut -d\  -f1 | rev | sed "s/)//"`
```

And when finishing, just attempting a `git checkout $branch` at the end.  If it was cleaned up, the git checkout should fail, and the behavior would be to stay in the main branch.

Sorry, I don't have time to PR this this weekend, so adding a feature request, and maybe I'll find time within the next couple of weeks to make a PR.

**Comment by davidraviv** [comment]
Thanks for the idea!
I'll work on it once I'll have a free time (hopefully very soon)

**Comment by davidraviv** [comment]
Implemented in v0.4
